Source/Graphics/OpenGL/GLRenderer2D.cpp

```diff
@@ -32,6 +32,8 @@ GLRenderer2D::GLRenderer2D() :
 	glGetIntegerv(GL_MAX_VERTEX_ATTRIBS, &m_MaxAttributes);
 	m_WantedAttributes = new unsigned char[m_MaxAttributes];
 	m_EnabledAttributes = new unsigned char[m_MaxAttributes];
+	std::memset(m_WantedAttributes, 0, m_MaxAttributes);
+	std::memset(m_EnabledAttributes, 0, m_MaxAttributes);
```

Comment:
Small cleanup so we're sure this is initialized properly and not filled with random junk.